### PR TITLE
Created rest endpoint on obs to service the registration of a bank account

### PR DESCRIPTION
**Pull request by Arielpetit**
Created rest endpoint on obs to service the registration of a bank account
[DESCRIPTION](https://github.com/orgs/ADORSYS-GIS/projects/14/views/4?sliceBy%5Bvalue%5D=Sprint1&pane=issue&itemId=83580595&issue=ADORSYS-GIS%7Cwebank-OnlineBanking%7C24)

### CODE REVIEW

File: obs/obs-rest-server/pom.xml [hunk 1]
```diff
@@ -32,6 +32,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.24</version> <!-- Check for the latest version -->
```
Comment (francis-pouatcha) [inline obs/obs-rest-server/pom.xml:38]: Please add lombok version to the parent property

File: online-banking-app/pom.xml [hunk 2]
```diff
@@ -61,6 +61,20 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.1.0</version>
```
Comment (francis-pouatcha) [inline online-banking-app/pom.xml:67]: Please send version number to top most parent pom. And declare dependency in the dependency management of that pom.

File: online-banking-app/src/test/java/com/adorsys/webank/OnlineBankingApplicationTests.java [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+//import org.springframework.boot.test.context.SpringBootTest;
+//
+//@SpringBootTest
+//class OnlineBankingApplicationTests {
+//
+//    @Test
+//    void contextLoads() {
+//        // This test will pass if the application context loads successfully
+//    }
+//}
```
Comment (francis-pouatcha) [inline online-banking-app/src/test/java/com/adorsys/webank/OnlineBankingApplicationTests.java:13]: Why not just delete the file?

**Review by francis-pouatcha — CHANGES_REQUESTED** [review]

**Review by NkwaTambe — APPROVED** [review]

File: tash apply
Comment (Koufan-De-King) [inline tash apply:1]: This file should be removed.

File: obs/obs-rest-api/pom.xml [hunk 4]
```diff
@@ -52,6 +52,11 @@
             <version>3.1.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (Koufan-De-King) [inline obs/obs-rest-api/pom.xml:55]: The version for this dependency should be derived from the parent pom.

**Review by Koufan-De-King — CHANGES_REQUESTED** [review]

**Review by Elwizzy12 — APPROVED** [review]

**Review by Koufan-De-King — DISMISSED** [review]
Great!

**Review by NkwaTambe — APPROVED** [review]

**Review by yvanhenang — DISMISSED** [review]

File: obs/obs-service-impl/pom.xml [hunk 5]
```diff
@@ -52,6 +52,12 @@
             </exclusions>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.adorsys.webank</groupId>
+            <artifactId>obs-rest-api</artifactId>
+            <version>0.1-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (francis-pouatcha) [inline obs/obs-service-impl/pom.xml:60]: remove this

**Review by francis-pouatcha — CHANGES_REQUESTED** [review]
Please see my comments

**Review by nancymuyeh — APPROVED** [review]

**Review by Koufan-De-King — APPROVED** [review]

**Review by Elwizzy12 — APPROVED** [review]

**Review by francis-pouatcha — COMMENTED** [review]
Good for me

**Pull request merged**